packages/desktop/scripts/create-build-dir.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs')
+
+const __SOURCE_DIRECTORY__ = 'public/build'
+const __DESTINATION_DIRECTORY__ = `${__SOURCE_DIRECTORY__}/build`
```

Comment:
Is there a reason why the `.node` files need to be in `public/build/build` instead of `public/build`?

Reply:
@rajivshah3 This should be removed (or fixed) as this is for some reason an issue that's specific to my local setup. It always try to look for the `.node` files in `public/build/build` instead of `public/build`

Reply:
I removed it!